README.md

```diff
@@ -54,28 +54,34 @@ docker compose exec dev format
+| `DATABASE_HOST`                   | The Postgres database host                                | _none_                                                      |
+| `DATABASE_PORT`                   | The Postgres database port                                | `5432`                                                      |
+| `CSP_IMG_SRC`                     | A comma separated list of CSP rules for `img-src`         | `'self'`                                                    |
+| `CSP_SCRIPT_SRC`                  | A comma separated list of CSP rules for `script-src`      | `'self'`                                                    |
+| `CSP_SCRIPT_SRC_ELEM`             | A comma separated list of CSP rules for `script-src-elem` | `'self'`                                                    |
+| `CSP_STYLE_SRC`                   | A comma separated list of CSP rules for `style-src`       | `'self'`                                                    |
+| `CSP_STYLE_SRC_ELEM`              | A comma separated list of CSP rules for `style-src-elem`  | `'self'`                                                    |
+| `CSP_FONT_SRC`                    | A comma separated list of CSP rules for `font-src`        | `'self'`                                                    |
+| `CSP_CONNECT_SRC`                 | A comma separated list of CSP rules for `connect-src`     | `'self'`                                                    |
+| `CSP_MEDIA_SRC`                   | A comma separated list of CSP rules for `media-src`       | `'self'`                                                    |
+| `CSP_WORKER_SRC`                  | A comma separated list of CSP rules for `worker-src`      | `'self'`                                                    |
+| `CSP_FRAME_SRC`                   | A comma separated list of CSP rules for `frame-src`       | `'self'`                                                    |
+| `GA4_ID`                          | The Google Analytics 4 ID                                 | _none_                                                      |
+| `ROSETTA_API_VERIFY_CERTIFICATES` | Verify certificate for API                                | `True`                                                      |
+| `ENVIRONMENT_NAME`                | The name of the environment (for reporting purposes)      | `production`                                                |
+| `SENTRY_DSN`                      | The ID of the Sentry client project to catch issues       | _none_                                                      |
+| `SENTRY_SAMPLE_RATE`              | How often to sample traces and profiles (0-1.0)           | production: `0.1`, staging: `0.25`, develop: `1`, test: `0` |
+| `BUILD_VERSION`                   | Generated in the CI/CD process                            | _none_                                                      |
```

Comment:
```suggestion
```

We don't need to document this because it's not an environment variable that can be changed by the application.

Reply:
Removed and add a comment in base.py, so that we know where its coming from.